finals-2024-01-16/code/4660-darko-ivanovski/src/app/services/data-map.ts

```diff
@@ -0,0 +1,44 @@
+import { Album, Band } from "./add-data"
```

Comment:
The point of the mapping is to transform the data to a better format. In this case both formats are the same, so the transformation is in place. Still, the code is ok, +5 points